packages/nimble-components/src/rich-text-editor/tests/rich-text-editor.spec.ts

```diff
@@ -10,4 +41,599 @@ describe('RichTextEditor', () => {
     it('should export its tag', () => {
         expect(richTextEditorTag).toBe('nimble-rich-text-editor');
     });
+
+    it('should initialize Tiptap editor', () => {
+        expect(pageObject.editorSectionHasChildNodes()).toBeTrue();
+        expect(pageObject.getEditorSectionFirstElementChildClassName()).toBe(
+            'ProseMirror'
+        );
+    });
+
+    it('should set aria role as "textbox"', () => {
+        const editor = element.shadowRoot?.querySelector('.editor');
+
+        expect(editor!.getAttribute('role')).toBe('textbox');
+    });
+
+    it('should set aria-multiline to true', () => {
+        const editor = element.shadowRoot?.querySelector('.editor');
+
+        expect(editor!.getAttribute('aria-multiline')).toBe('true');
+    });
+
+    it('should have a slot and part named "footer-actions"', () => {
```

Comment:
This test and the page object methods it uses seem overly detailed to me. The contract that we want to verify is less about the existence of this slot and part and more about content that is placed in that slot being positioned correctly. I think that's covered by the matrix test so I'd vote to delete this and the page object methods it calls. 

You could instead consider adding unit tests similar to how a client would verify its slotted content behaves correctly. For example, write a test that queries for and clicks the OK or cancel button. But that likely wouldn't need these same page object methods either.

Reply:
I have removed these test cases and added a test as you suggested which checks the click event works as expected for the slotted buttons.